netkan/setup.py

```diff
@@ -30,7 +31,6 @@
     },
     extras_require={
         'development': [
-            'pytest',
             'ptvsd',
```

Comment:
Have you checked if VS Code uses this? (I don't think it does, I'll have to check).

Reply:
`.vscode/settings.json` contains the following lines:
```json
  "python.testing.pytestEnabled": false,
  "python.testing.nosetestsEnabled": false,
  "python.testing.unittestEnabled": true,
```

So VSCode shouldn't use pytest. Additionally, all our tests are set up for `unittest`, I don't know if `pytest` could even run them.

However I can't test it, I don't use VS Code.

Reply:
Nope, not required by VS Code by the looks. PyTest _can_ run unittest TestCase stuff, but we can worry about it if we want to use it later.

> pytest supports running Python unittest-based tests out of the box. It’s meant for leveraging existing unittest-based test suites to use pytest as a test runner and also allow to incrementally adapt the test suite to take full advantage of pytest’s features.